packages/svelte/src/runtime/tag-version.js

```diff
@@ -0,0 +1,5 @@
+import { VERSION } from '../shared/version.js';
+
+if (typeof window !== 'undefined')
+	// @ts-ignore
+	(window.__svelte_versions || (window.__svelte_versions = [])).push(VERSION);
```

Comment:
In case we want to add more to the global in the future (who knows), would it be better to go back to the previous iteration where it was a `version` prop on an object?

Reply:
Set is unnecessary overhead considering there'll usually only be one version.

> where it was a version prop on an object
version prop but with an array value? What should the variable be called?

Reply:
`versions`? 😅 `Set` is _probably_ overhead for only on version, but if it's only one version it won't be any overhead, and if it's called so often that it matters it's likely there are many duplicate versions. I just want to avoid an arcane bug due to someone having a weird setup with thousands of self-contained JS files which all append to that array. I'm also fine with going back to "first/last one wins"